usecase/zipcodes.go

```diff
@@ -0,0 +1,183 @@
+package usecase
+
+import (
+	"context"
+	"encoding/csv"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/AlonSerrano/GolangBootcamp/models"
+
+	"github.com/google/uuid"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"golang.org/x/text/encoding/charmap"
+)
+
+var rows = []map[string]string{
+	{"isoCode": "MX-AGU"},
+	{"isoCode": "MX-BCN"},
+	{"isoCode": "MX-BCN"},
+	{"isoCode": "MX-BCS"},
+	{"isoCode": "MX-CAM"},
+	{"isoCode": "MX-COA"},
+	{"isoCode": "MX-COL"},
+	{"isoCode": "MX-CHP"},
+	{"isoCode": "MX-CHH"},
+	{"isoCode": "MX-CMX"},
+	{"isoCode": "MX-DUR"},
+	{"isoCode": "MX-GUA"},
+	{"isoCode": "MX-GRO"},
+	{"isoCode": "MX-HID"},
+	{"isoCode": "MX-JAL"},
+	{"isoCode": "MX-MEX"},
+	{"isoCode": "MX-MIC"},
+	{"isoCode": "MX-MOR"},
+	{"isoCode": "MX-NAY"},
+	{"isoCode": "MX-NLE"},
+	{"isoCode": "MX-OAX"},
+	{"isoCode": "MX-PUE"},
+	{"isoCode": "MX-QUE"},
+	{"isoCode": "MX-ROO"},
+	{"isoCode": "MX-SLP"},
+	{"isoCode": "MX-SIN"},
+	{"isoCode": "MX-SON"},
+	{"isoCode": "MX-TAB"},
+	{"isoCode": "MX-TAM"},
+	{"isoCode": "MX-TLA"},
+	{"isoCode": "MX-VER"},
+	{"isoCode": "MX-YUC"},
+	{"isoCode": "MX-ZAC"},
+}
```

Comment:
I kind of remember telling you that a map would be better here but seems that I was wrong and in reality, an array would be the best option, also the name `rows` should be changed to something more descriptive.